packages/core/contracts/merkle-distributor/implementation/MerkleDistributor.sol

```diff
@@ -184,9 +166,7 @@ contract MerkleDistributor is Ownable, Testable {
         // Check the account has not yet claimed for this window.
         require(!claimed[claim.windowIndex][claim.account], "Account has already claimed for this window");
 
-        // Proof is correct and claim has not occurred yet; check that claim window has begun.
-        require(getCurrentTime() >= merkleWindows[claim.windowIndex].start, "Claim window has not begun");
-
+        // Proof is correct and claim has not occurred yet, mark claimed complete.
         claimed[claim.windowIndex][claim.account] = true;
         emit Claimed(
             msg.sender,
```

Comment:
unrelated to this line, but I think we can drop this if statement below to kill the last TODO in this contract :) :
```
// TODO: Should we revert claims for 0 tokens?
if (amount > 0)
```

We shouldn't be putting 0-token distributions in there anyway, and most tokens don't revert on 0-token transfers, so I would argue just to drop the if statement and blindly transfer. The only way I could imagine the user getting there is passing 0 claims into the batcher, which IMO isn't a big concern and easy to diagnose.